src/main/java/com/endava/soa_5ed/task2/regex/StringGenerator.java

```diff
@@ -0,0 +1,54 @@
+package com.endava.soa_5ed.task2.regex;
+
+public class StringGenerator {
+    /* Date Time and Regex - Exercise 1 */
+    final static String baseStr = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890,./;'[]\\<>?:\"{}|`~!@#$%^&*()_+-=";
+    int baseStrLen = baseStr.length();
+
+    public String getGenStr(int genStrLen) {
+        return generateString(genStrLen);
+    }
+
+    public String getNoNumStr(String string) {
+        return removeNumChar(string);
+    }
+
+    public String getNoAlphaStr(String string) {
+        return removeAlphaChar(string);
+    }
+
+    public String getNoSymbolStr(String string) {
+        return removeSymbols(string);
+    }
+
+    public void printMessage(String genString) {
+        printMsg(genString);
+    }
+
+    private String generateString(int genStrLen) {
+        StringBuilder builtString = new StringBuilder();
+        for (int i = 0; i < genStrLen; i++) {
+            builtString.append(baseStr.charAt((int) (Math.random() * (baseStrLen - 1))));
+        }
+        return builtString.toString();
+    }
+
+    private String removeNumChar(String genStr) {
+        return genStr.replaceAll("[0-9]", "");
+    }
+
+    private String removeAlphaChar(String genStr) {
+        return genStr.replaceAll("[a-z]|[A-Z]", "");
+    }
+
+    private String removeSymbols(String genStr) {
+        return genStr.replaceAll("[/[^\\p{L}\\d\\s]u]", "");
+    }
+
+    private void printMsg(String string) {
```

Comment:
You may just override .toString() 
and return the text from sout input.

Reply:
I've changed some code and this isn't needed anymore